OpenSourceOrg/dotOrg
# Add ai.php from staging to production

#### Changes proposed in this Pull Request

Add `ai.php` file to production. Copied over from a staging site.

#### Testing instructions



* Test to make sure that the template file behaves as expected in the branch `/add/ai-php`.

Resolves: #139
